CHANGELOG.md

```diff
@@ -25,7 +25,7 @@ We’ve also made fixes in the following pull requests:
 
 - [#2549: Fix header with product name focus and hover state length](https://github.com/alphagov/govuk-frontend/pull/2549)
 - [#2573: Better handle cases where `$govuk-text-colour` is set to a non-colour value](https://github.com/alphagov/govuk-frontend/pull/2573)
-- [#2590: Maxlength attribute should not be present on textarea after character-count JavaScript has kicked in](https://github.com/alphagov/govuk-frontend/pull/2590)
+- [#2590: Remove `maxlength` attribute from `textarea` after character count JavaScript has started to work](https://github.com/alphagov/govuk-frontend/pull/2590)
```

Comment:
@EoinShaughnessy Do you think this would be more accurate or more confusing for people reading the release notes?

```suggestion
- [#2590: Remove `maxlength` attribute from `textarea` after character count JavaScript has been initialised](https://github.com/alphagov/govuk-frontend/pull/2590)
```

Reply:
@vanitabarrett Sounds good!